generator.go

```diff
@@ -0,0 +1,48 @@
+package justgo
+
+import (
+	"embed"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/gophers-mx/just-go/config"
+	"github.com/gophers-mx/just-go/pkg"
```

Comment:
why putting everything into `pkg`? I'd rather throw content into proper subfolders, under `pkg` of course.